gnovm/stdlibs/html/entity.gno

```diff
@@ -0,0 +1,2263 @@
+// Copyright 2010 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package html
+
+// All entities that do not end with ';' are 6 or fewer bytes long.
+const longestEntityWithoutSemicolon = 6
+
+// entity is a map from HTML entity names to their values. The semicolon matters:
+// https://html.spec.whatwg.org/multipage/named-characters.html
+// lists both "amp" and "amp;" as two separate entries.
+//
+// Note that the HTML5 list is larger than the HTML4 list at
+// http://www.w3.org/TR/html4/sgml/entities.html
+var entity map[string]rune
+
+// HTML entities that are two unicode codepoints.
+var entity2 map[string][2]rune
+
+// populateMapsOnce guards calling populateMaps.
+var populateMapsOnce bool
```

Comment:
Can we remove this bool and check by the map size?

Reply:
Or maybe you can populate these maps directly in the declaration? It is guaranteed to be initialized only once.
Why is `populateMaps` necessary ?

Reply:
you're right; this is the kind of optimization that somewhat makes sense for go but i think it's fine if we discard for gno.

so let's just have these in the declarations directly.

Reply:
Sure thanks @petar-dambovaliev @thehowl  for the review indeed it is not required 👍 
I removed the function and just initialize directly both tables on a5f4a1b4f6f5583200a3ff0ce9cd1f7274173513